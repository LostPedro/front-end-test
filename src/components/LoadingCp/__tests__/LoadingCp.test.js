import React from 'react'
import {render, cleanup} from '@testing-library/react'
import {LoadingCp} from '../'

afterEach(cleanup)

it('should take a snapshot', () => {
  const {asFragment} = render(<LoadingCp />)

  expect(asFragment(<LoadingCp />)).toMatchSnapshot()
})
