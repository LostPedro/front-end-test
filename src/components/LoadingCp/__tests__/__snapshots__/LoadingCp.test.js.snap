// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`should take a snapshot 1`] = `
<DocumentFragment>
  <div
    class="loading-component-fade"
    style="background-color: rgba(0, 0, 0, 0.75);"
  >
    <div
      class="ant-spin ant-spin-lg ant-spin-spinning"
    >
      <span
        class="ant-spin-dot ant-spin-dot-spin"
      >
        <i
          class="ant-spin-dot-item"
        />
        <i
          class="ant-spin-dot-item"
        />
        <i
          class="ant-spin-dot-item"
        />
        <i
          class="ant-spin-dot-item"
        />
      </span>
    </div>
  </div>
</DocumentFragment>
`;
